doc/whatsnew/fragments/7281.new_check

```diff
@@ -0,0 +1,3 @@
+Add ``magic-number`` plug-in checker for comparison with constants instead of literals.
```

Comment:
```suggestion
Add ``magic-number`` plug-in checker for comparison with constants instead of literals.
You can use it with ``--load-plugins=pylint.extensions.magic_value``
```